tests/network/test_services.py

```diff
@@ -43,9 +47,11 @@ class MockBackend(NetworkAPI):
     IGNORED_ERRORS = NetworkAPI.IGNORED_ERRORS
     GET_BALANCE_MAIN = [raise_connection_error]
     GET_TRANSACTIONS_MAIN = [raise_connection_error]
+    GET_TRANSACTIONS_BY_ID_MAIN = [raise_connection_error]
     GET_UNSPENT_MAIN = [raise_connection_error]
     GET_BALANCE_TEST = [raise_connection_error]
     GET_TRANSACTIONS_TEST = [raise_connection_error]
+    GET_TRANSACTIONS_BY_ID_TEST = [raise_connection_error]
```

Comment:
Should be `GET_TRANSACTION_BY_ID_TEST` (without an `S` after `TRANSACTION`) as defined in `NetworkAPI` inside `bit/network/services.py`.

Reply:
Done, good catch.